examples/python/arkit_scenes/main.py

```diff
@@ -254,8 +254,7 @@ def log_camera(
     # pathlib makes it easy to get the parent, but log methods requires a string
     camera_path = str(PosixPath(entity_id).parent)
     rr.log_transform3d(camera_path, camera_from_world, from_parent=True)
-    rr.log_view_coordinates(camera_path, xyz="RDF")  # X=Right, Y=Down, Z=Forward
-    rr.log_pinhole(f"{entity_id}", child_from_parent=intrinsic, width=w, height=h)
+    rr.log_pinhole(entity_id, width=w, height=h, child_from_parent=intrinsic)
```

Comment:
I'm not sure.

We can either add `camera_xyz=RDF` to all examples to be explicit. This could either be taken as an encouragement to users to use RDF (good), or as an invitation to change it to something else (perhaps less good).

If we instead omit it when not needed it could lead to questions of what the default is and how to change it.